hermes/src/ontology/mod.rs

```diff
@@ -22,8 +12,43 @@ pub use self::nlu::*;
 pub use self::tts::*;
 pub use self::vad::*;
 
+pub mod asr;
+pub mod audio_server;
+pub mod dialogue;
+pub mod hotword;
+pub mod injection;
+pub mod nlu;
+pub mod tts;
+pub mod vad;
+
 pub trait HermesMessage<'de>: fmt::Debug + Deserialize<'de> + Serialize {}
 
+#[derive(Serialize, Deserialize, PartialEq, PartialOrd, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub enum SnipsComponent {
+    Asr,
+    Nlu,
+    Dialogue,
+    Hotword,
+    Injection,
+    Tts,
+    AudioServer,
+}
+
+impl fmt::Display for SnipsComponent {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            SnipsComponent::Asr => write!(f, "Asr"),
+            SnipsComponent::Nlu => write!(f, "Nlu"),
+            SnipsComponent::Dialogue => write!(f, "Dialogue"),
+            SnipsComponent::Hotword => write!(f, "Hotword"),
+            SnipsComponent::Injection => write!(f, "Injection"),
+            SnipsComponent::Tts => write!(f, "Tts"),
+            SnipsComponent::AudioServer => write!(f, "Audio Server"),
```

Comment:
Where is this needed ? shouldn't the `Debug` impl be enough ?  Having a space here in `Audio Server` is a timebomb

Reply:
We implemented this trait because it was _"needed"_ for `snips-watch`. We can get rid of this.

Reply:
It's not needed anymore